### PR TITLE
Dnf package

### PR DESCRIPTION
**Pull request by brewn**

### CODE REVIEW

**Comment by brewn** [comment]
#628

File: chef_master/source/resource_dnf_package.rst [hunk 1]
```diff
@@ -0,0 +1,471 @@
+=====================================================
+dnf_package
+=====================================================
+`[edit on GitHub] <https://github.com/chef/chef-web-docs/blob/master/chef_master/source/resource_dnf_package.rst>`__
+
+.. tag resource_package_dnf
+
+Use the **dnf_package** resource to install, upgrade, and remove packages with dnf for Fedora platforms. The **dnf_package** resource is able to resolve ``provides`` data for packages much like dnf can do when it is run from the command line. This allows a variety of options for installing packages, like minimum versions, virtual provides, and library names.
```
Comment (tas50) [inline chef_master/source/resource_dnf_package.rst]: DNF should be capitalized

File: chef_master/source/resource_dnf_package.rst [hunk 2]
```diff
@@ -0,0 +1,471 @@
...
+     provider                   Chef::Provider::Package::dnf
+     retries                    Integer
+     retry_delay                Integer
+     sensitive                  TrueClass, FalseClass # defaults to ``false``
+     source                     String
+     subscribes                 # see description
+     timeout                    String, Integer
+     version                    String, Array
+     action                     Symbol # defaults to :install if not specified
+
```
Comment (tas50) [inline chef_master/source/resource_dnf_package.rst]: This extra line should go away

File: chef_master/source/resource_dnf_package.rst [hunk 3]
```diff
@@ -0,0 +1,471 @@
...
+A resource may specify multiple packages and/or versions for platforms that use Yum, DNF, Apt, Zypper, or Chocolatey package managers. Specifing multiple packages and/or versions allows a single transaction to:
+
+* Download the specified packages and versions via a single HTTP transaction
+* Update or install multiple packages with a single resource during the chef-client run
+
+For example, installing multiple packages:
+
+.. code-block:: ruby
+
+   package ['package1', 'package2']
```
Comment (tas50) [inline chef_master/source/resource_dnf_package.rst]: We should update this to a cookstyle compliant setup:

package %w(package1 package2)

**Comment by brewn** [comment]
@lamont-granquist going to note that this doesn't include the Provider change we discussed - that's going to be applied separately in a different PR.

**Review by tas50 — APPROVED** [review]

**Review by lamont-granquist — APPROVED** [review]

**Pull request merged**